<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YtSettings</class>
 <widget class="QDialog" name="YtSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Youtube Download Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="sliderQuality">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboAudioExtention">
       <item>
        <property name="text">
         <string>m4a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>aac</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>flac</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mp3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>opus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vorbis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>wav</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboVideoExtention">
       <item>
        <property name="text">
         <string>mp4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>avi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>flv</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ogg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>webm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mkv</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineExec">
       <property name="text">
        <string>/bin/youtube-dl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolExecutable">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkMetadata">
       <property name="text">
        <string>Add Metadata</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkDebug">
       <property name="text">
        <string>Enable Debug</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkThumbnail">
       <property name="text">
        <string>Add Thumbnail</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkAskAlways">
       <property name="text">
        <string>Ask always settings</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkSubtitle">
       <property name="text">
        <string>Add Subtitle (ENG)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkExtract">
       <property name="text">
        <string>Extract Audio</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkAskAlwaysFile">
       <property name="text">
        <string>Ask always output filename</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkProxy">
       <property name="text">
        <string>Use Proxy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineDefaultDir"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
